doc/TILESET.md

```diff
@@ -182,17 +182,30 @@ A tilesheet can be an expansion from a mod.  Each expansion tilesheet is a singl
 
 `--palette-copies`: Output copies of tilesheet files quantized to 8bpp palette with `.png8` extension. Intended for external detection if conversion was lossless.
 
-Requires `pyvips` module.
+Requires `pyvips` module, here is how you can install it on Windows:
 
-### Installing pyvips on Windows
```

Comment:
Doesn't that make it a bit less obvious that this is only for windows? Or

Reply:
Is this better?

Reply:
Yes, nice 👍